src/commands/a32nx/remoteMcdu.ts

```diff
@@ -0,0 +1,17 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+export const remoteMcdu: CommandDefinition = {
+    name: ['remotemcdu', 'remote'],
```

Comment:
I may be inclined to remove the `remote` alias for when other remote features are implemented? Perhaps remote could end up being a generalised command later on